jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopoverCell.java

```diff
@@ -45,84 +43,88 @@
 
     @Inject
     @DataField("popover")
-    private Popover popover;
```

Comment:
@rbarriuso exactly, the previous approach only really worked for `Anchor` elements, but if you look at the JavaScript API that isn't true, basically any element could have a popover.